modules/cookbooks/pages/kafka_migrator.adoc

```diff
@@ -0,0 +1,352 @@
+= Kafka Migrator
+
+// tag::single-source[]
+
+Author: https://www.linkedin.com/in/mtodor[Mihai Todor^]
```

Comment:
```suggestion
```

This isn't something we add to our docs. We tend to do this for blog posts.

Reply:
Oops, leftover from the blog post. Thank you!